src/keploy.ts

```diff
@@ -76,6 +77,25 @@ export default class Keploy {
     this.responses = {};
     this.dependencies = {};
     this.mocks = {};
+    if (
+      process.env.KEPLOY_MODE === "record" ||
+      process.env.KEPLOY_MODE === "test"
+    ) {
+      exec("which keploy", (error, stdout, stderr) => {
+        if (!stdout) {
+          console.log(
+            "Keploy Server is not installed. Please install it using https://docs.keploy.io/docs/go/installation "
+          );
+          process.exit();
+        } else {
+          exec("lsof -i:6789", (error, stdout, stderr) => {
+            if (!stdout) {
+              spawn("keploy");
```

Comment:
Here, run the Keploy binary with `KEPLOY_PORT=6789` env variable.

Reply:
So I should use the command `export KEPLOY_PORT=6789` before spawning the keploy binary right?

Reply:
Yes, this export cmd can be run with keploy using ` && ` in the exec method

Reply:
I have done the changes please have a look :sparkles:

Reply:
LGTM. Please rebase your commit messages for the github action to pass

Reply:
I have made the changes to commit messages. I didn't know there was a specific way to write the commit messages. Thank you for being so patient :sweat_smile: